jdbc/pom.xml

```diff
@@ -189,17 +235,24 @@
                         <include>org/apache/hadoop/security/*</include>
                         <include>org/apache/hadoop/conf/*</include>
                         <include>org/apache/hadoop/fs/*</include>
-                        <include>org/apache/hadoop/metrics2/*</include>
-                        <include>org/apache/hadoop/metrics2/lib/*</include>
-                        <include>org/apache/hadoop/metrics2/impl/*</include>
-                        <include>org/apache/hadoop/util/ReflectionUtils.class</include>
-                        <include>org/apache/hadoop/util/ReflectionUtils$1.class</include>
+                        <include>org/apache/hadoop/fs/impl/*</include>
+                        <include>org/apache/hadoop/fs/statistics/*</include>
+                        <include>org/apache/hadoop/metrics2/**</include>
+                        <include>org/apache/hadoop/security/token/**</include>
+                        <include>org/apache/hadoop/util/*</include>
                         <include>org/apache/hadoop/io/Writable.class</include>
-                        <include>org/apache/hadoop/io/DataOutputBuffer.class</include>
-                        <include>org/apache/hadoop/io/DataInputBuffer.class</include>
+                        <include>org/apache/hadoop/net/*</include>
+                        <include>org/apache/hadoop/io/*</include>
                         <include>org/apache/hadoop/HadoopIllegalArgumentException.class</include>
                       </includes>
                     </filter>
+                    <filter>
+                      <artifact>org.apache.hadoop:hadoop-auth</artifact>
```

Comment:
previously it was excluded completely, why it that change?

Reply:
This is for UGI to login the user from keytab, 
```
Exception in thread "main" java.lang.NoClassDefFoundError: org/apache/hadoop/util/PlatformName
	at org.apache.hadoop.security.UserGroupInformation.getOSLoginModuleName(UserGroupInformation.java:434)
	at org.apache.hadoop.security.UserGroupInformation.<clinit>(UserGroupInformation.java:462)
	at org.example.App.main(App.java:45)